### PR TITLE
add api tests

### PR DESCRIPTION
**Pull request by dziana-babrova**

### CODE REVIEW

File: tests/get-scenario.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+});
+
+it("the value of the content-type header is application/json; charset=utf-8", async () => {
+    chai.expect(response.headers["content-type"]).to.be.equal("application/json; charset=utf-8");
+});
+
+it("the content of the response body is the array of 10 users", async () => {
+    chai.expect(response.data).to.be.array();
+    chai.expect(response.data).to.be.ofSize(10);
+});
```
Comment (rimus) [inline tests/get-scenario.js]: I hope these sections should be inside describe :) Please check whole PR.

File: tests/post-scenario.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+it("the value of the content-type header is application/json; charset=utf-8", async () => {
+  chai.expect(response.headers["content-type"]).to.be.equal("application/json; charset=utf-8");
+});
+
+it("the content of the response body is the array of 10 users", async () => {
+  chai.expect(response.data.title).to.be.equal("foo");
+    chai.expect(response.data.body).to.be.equal("bar");
+    chai.expect(response.data.userId).to.be.equal(1);
+    chai.expect(response.data.id).to.exist;
```
Comment (rimus) [inline tests/post-scenario.js:32]: Format the code.

**Review by rimus — CHANGES_REQUESTED** [review]

File: tests/post-scenario.js (hunk 2 above)
Comment (dziana-babrova) [reply tests/post-scenario.js:32]: fixed

**Review by rimus — APPROVED** [review]
Looks good.

**Pull request merged**